specification/signalr/resource-manager/Microsoft.SignalRService/preview/2025-01-01-preview/signalr.json

```diff
@@ -2,92 +2,138 @@
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
   },
+  "tags": [
+    {
+      "name": "Operations"
+    },
+    {
+      "name": "SignalRResources"
+    },
+    {
+      "name": "CustomCertificates"
+    },
+    {
+      "name": "CustomDomains"
+    },
+    {
+      "name": "PrivateEndpointConnections"
+    },
+    {
+      "name": "Replicas"
+    },
+    {
+      "name": "SharedPrivateLinkResources"
+    },
+    {
+      "name": "SignalRSharedPrivateLinkResources"
+    }
+  ],
   "paths": {
     "/providers/Microsoft.SignalRService/operations": {
       "get": {
-        "tags": [
-          "SignalR"
-        ],
-        "description": "Lists all of the available REST API operations of the Microsoft.SignalRService provider.",
         "operationId": "Operations_List",
-        "produces": [
-          "application/json"
+        "tags": [
+          "Operations"
         ],
+        "description": "List the operations for the provider",
         "parameters": [
           {
             "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
           }
         ],
         "responses": {
           "200": {
-            "description": "Success. The response describes the list of operations.",
+            "description": "The request has succeeded.",
             "schema": {
               "$ref": "#/definitions/OperationList"
             }
           },
           "default": {
-            "description": "Error response describing why the operation failed.",
+            "description": "An unexpected error response.",
             "schema": {
               "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
             }
           }
         },
-        "x-ms-pageable": {
-          "nextLinkName": "nextLink"
-        },
         "x-ms-examples": {
           "Operations_List": {
             "$ref": "./examples/Operations_List.json"
           }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
         }
       }
     },
     "/subscriptions/{subscriptionId}/providers/Microsoft.SignalRService/locations/{location}/checkNameAvailability": {
       "post": {
-        "tags": [
-          "SignalR"
-        ],
-        "description": "Checks that the resource name is valid and is not already in use.",
         "operationId": "SignalR_CheckNameAvailability",
-        "produces": [
-          "application/json"
-        ],
+        "description": "Checks that the resource name is valid and is not already in use.",
         "parameters": [
           {
-            "in": "path",
-            "name": "location",
-            "description": "The region",
-            "required": true,
-            "type": "string"
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
```

Comment:
The parameter order has been changed. Isn't that a breaking change?
Should we keep the original order?

Reply:
It is not on REST API level. The path is on URL, and the URL would be same, regardless of the order of Swagger parameter definition.